package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "primo-studio-browzine",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "Primo Studio BrowZine addon",
```

Comment:
Any idea if we want to change the name to include the word "LibKey"? Guessing not, and it'll match the keywords, but thought I'd ask just in case.

Reply:
> Any idea if we want to change the name to include the word "LibKey"? Guessing not, and it'll match the keywords, but thought I'd ask just in case.

That's a great question @karlbecker! 

Thoughts on this @johnseguin?

Reply:
@tiandavis the name in the store right now appears like this:

![image](https://user-images.githubusercontent.com/1850626/77112449-07fbc300-69f7-11ea-93d1-a80d03e24d42.png)

So I'd prefer if we could do something similar here for consistency like "Primo Studio LibKey|BrowZine addon" etc.  Thanks!

Reply:
Yikes! Looks like there are two instances in there for some reason:
http://primo-studio.exlibrisgroup.com/?url=https:%2F%2Fsandbox01-na.primo.exlibrisgroup.com&vid=01COLSCHL_INST:BRZNTEST&ve=true&dirName=1SwAav&central=false

![image](https://user-images.githubusercontent.com/376230/77113530-6463e000-6a01-11ea-9018-5c7ae9e47590.png)

Reply:
@tiandavis Yes please. :)

Reply:
Will do!